apisix/plugins/request-id.lua

```diff
@@ -202,14 +215,28 @@ local function next_id()
     return snowflake:next_id()
 end
 
+-- generate random id
+local function get_random_id(strs, length)
+    local res = ""
+    for i = 1,length do
+        local idx = math_random(str_len(strs))
+        res = res .. str_sub(strs, idx, idx)
+    end
+    return res
+end
```

Comment:
I provide a faster implementation:
```
local function get_random_id(strs, length)
    local res = ffi.new("unsigned char[?]", length)
    for i = 0, length - 1 do
        res[i] = string.byte(strs, math_random(#strs))
    end
    return ffi.string(res, length)
end
```

You can write a benchmark to see the difference.